src/Settings/VideoSettings.h

```diff
@@ -35,6 +35,16 @@ class VideoSettings : public SettingsGroup
     DEFINE_SETTINGFACT(streamEnabled)
     DEFINE_SETTINGFACT(disableWhenDisarmed)
     DEFINE_SETTINGFACT(lowLatencyMode)
+    DEFINE_SETTINGFACT(forceVideoDecoder)
+
+    enum VideoDecoderOptions {
+        Default = 0,
```

Comment:
Can you prefix all of these with `ForceVideoDecoder`. Referencing like `VideoSettings::Default` is just too generic to mean much, whereas `VideoSettings::ForceVideoDecoderDefault` is understandable at a glance.

Reply:
Using ForceVideoDecoder prefix now.